test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -58,7 +60,13 @@ public void VerifyImages(ImageDescriptor imageDescriptor)
 
             try
             {
-                VerifySdkImage_NewRestoreRun(imageDescriptor, appSdkImage);
+                CreateTestAppWithSdkImage(imageDescriptor, appSdkImage);
+
+                if (!String.Equals("arm", imageDescriptor.Architecture, StringComparison.OrdinalIgnoreCase))
```

Comment:
Should make use of the IsArm property on imageDescriptor.

Reply:
Good catch!

Reply:
Updated.